### DISCUSSION TITLE
Add SensorDeviceClass.CONDUCTIVITY

### DISCUSSION THREAD

**Discussion by dontinelli** [home-assistant/architecture#1095] [Entity Models] [answered]
Currently HA provides only one unit for conductivity, namely Microsiemens. The units should be extended to Millisiemens and Siemens and a respective SensorDeviceClass.CONDUCTIVITY in order enable also the conversion between those unities (cf. https://github.com/home-assistant/core/pull/118728 for details for implementation).

**Comment by joostlek** [comment]
Please add a list of core integrations that can use this right now

**Reply by NECH2004** [reply]
mysensors, plant, xiaomi_ble

**Reply by dontinelli** [reply]
> mysensors, plant, xiaomi_ble

And fyta

**Comment by NECH2004** [comment]
I think it is a good and valid suggestion.
If you look at the typical conductivity values ​​for soils or river water for irrigation, the conductivity is often around 1 mS/cm (= dS/m).

From a more Physics view I think that the base SI units (here S/m) should also (always) be available.

**Comment by frenck** [comment, marked as answer]
We have discussed this proposal with the core team, and approved it.

This can be implemented / added to the core.

../Frenck

**Comment by frenck** [comment]
Change has been merged into current, closing the arch discussion for that reason.

../Frenck
